crates/services/p2p/src/behavior.rs

```diff
@@ -46,7 +46,7 @@ const MAX_PENDING_OUTGOING_CONNECTIONS: u32 = 100;
 /// Handles all p2p protocols needed for Fuel.
 #[derive(NetworkBehaviour)]
 pub struct FuelBehaviour {
-    // **WARNING**: The order of the behaviours is important and fragile, at least for the tests.
+    /// **WARNING**: The order of the behaviours is important and fragile, at least for the tests.
```

Comment:
does this have to be a comment that shows in documentation? afaiu its just a dev comment that tells someone modifying this crate not to reorder things